src/rp2_common/hardware_rcp/include/hardware/rcp.h

```diff
@@ -15,13 +15,15 @@
  */
 
 // ----------------------------------------------------------------------------
-// RCP instructions (this header is Arm-only)
-#if defined(PICO_RP2350) && !defined(__riscv)
+// RCP masks (this header is RP2350 only)
+#if defined(PICO_RP2350)
```

Comment:
lets make these (and the ones below) `#if HAS_REDUNDANCY_COPROCESSOR`

Reply:
This one needs to be defined for Risc-V too, so I've left it as `#if PICO_RP2350` - I've made the one below `#if HAS_REDUNDANCY_COPROCESSOR` as that is still Arm only

Reply:
ok, makes sense. it was a `PICO_RP2350` check before, but lets make this a `#if !PICO_RP2040`